pyproject.toml

```diff
@@ -5,15 +5,18 @@ requires = [
     "ninja",
     "packaging",
     "setuptools >= 49.4.0",
-    "torch == 2.1.2",
+    "torch == 2.2.1",
```

Comment:
note that we will have to release a nightly version of magic-wand with this torch support to get testing working for this PR